src/features/practice/ui/TestInterface.tsx

```
@@ -236,124 +224,135 @@ const TestInterface = () => {
+                      <div className="flex items-center gap-4">
+                        <Badge variant="outline" className="text-lg font-medium">{currentQuestion.subject}</Badge>
+                        <div className="flex items-center gap-2 bg-muted p-2 rounded-md">
+                          <Clock className="h-5 w-5 text-muted-foreground" />
+                          <Countdown date={effectiveEndTime} renderer={CountdownRenderer} onComplete={handleCountdownComplete} />
+                        </div>
+                      </div>
+                    </CardTitle>
+                  </CardHeader>
+                  <CardContent>
+                    <div className="space-y-6">
+                      {currentQuestion.section && <div className="text-sm font-medium text-muted-foreground mb-4" dangerouslySetInnerHTML={createMarkup(currentQuestion.section)} />}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)